### ISSUE TITLE
The Attack quest is bugged

### ISSUE THREAD

**Issue by smichael1** [dalaranwow/dalaran-wow#5092]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Quest: The Attack does not wait for the conversation to complete before the 'overhear conversation' goal of the quest is met.  In addition, only the group member that started the quest ends up hearing the conversation.
**Expected behaviour**: Tell us what should happen instead
The entire conversation should take at least 10-20 seconds to complete and at that point all group members close to the conversation should complete the 'overhear conversation' goal of the quest.
**Steps to reproduce the problem**:
Start the quest with a group and wait in the courtyard
1. 
2. 
3. 

**Include proofs for this behaviour**
Our party of 3 had to do the quest three times for us all to complete it.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @smichael1.
Both issues confirmed on PTR.
I would like to add a 3-rd one also:
 - $n in the text:
![wowscrnshot_112217_124608](https://user-images.githubusercontent.com/6797611/33124117-c680fa58-cf84-11e7-94e9-d57146749a63.jpg)
